task/t2.py

```diff
@@ -0,0 +1,25 @@
+from utils.read_files import read_files_t2
+
+file_values = read_files_t2()
+
+temperatures = []
+for file_value in file_values:
+    if file_value and len(file_value) >= 4 and file_value[1] and file_value[3]:
```

Comment:
good, now dont use chat gpt and write your own code

Reply:
sure.